src/ucp/wireup/select.c

```diff
@@ -1721,30 +1720,37 @@ ucp_wireup_add_am_bw_lanes(const ucp_wireup_select_params_t *select_params,
 
     bw_info.local_dev_bitmap  = UINT64_MAX;
     bw_info.remote_dev_bitmap = UINT64_MAX;
-    bw_info.max_lanes         = context->config.ext.max_eager_lanes - 1;
-    /* rndv/am/zcopy proto should take max_rndv_lanes value into account */
-    if (context->config.ext.proto_enable) {
-        bw_info.max_lanes = ucs_max(bw_info.max_lanes,
-                                    context->config.ext.max_rndv_lanes - 1);
-    }
+    excluded_am_lane          = UCP_NULL_LANE;
```

Comment:
shall we also update construct_lanes?
https://github.com/openucx/ucx/blob/master/src/ucp/wireup/select.c#L2394